File: Server-Spring-Java/src/main/java/pyo/quizgame/dto/QuizInfoDto.java
package pyo.quizgame.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class QuizInfoDto {
    private String title;
    private String example1;
    private String example2;
    private String example3;
    private String example4;
    private String filePath;
    private Long answer;
}

File: Front-Android-Kotlin/app/src/main/java/com/pyo/quizfrontapp/data/quizInfoDto.kt
package com.pyo.quizfrontapp.data

data class quizInfoDto(
    var title:String,
    var example1:String,
    var example2:String,
    var example3:String,
    var example4:String,
    var filePath:String,
    var answer:Int,
    var selectedAnswer:Long,
)
